docs/getting-started.md

```diff
@@ -0,0 +1,138 @@
+## Getting Started
+
+### Prerequisites
+The following dependencies are needed to compile/build/run.
+* gcc/g++ >=4.9 ( std=c++14 is used )
+* meson >= 0.50
+* libopenblas-dev and base
+* tensorflow-lite >=1.14.0
+* libiniparser
+* libjsoncpp >=0.6.0 ( if you wand to use open AI )
+* libcurl3 >=7.47 ( if you wand to use open AI )
+* libgtest ( for testing )
+
+### Install via PPA repository (Debian/Ubuntu)
+
+The nntrainer releases are at a PPA repository. In order to install it, use:
+
+```bash
+$ sudo apt-add-repository ppa:nnstreamer
+$ sudo apt install nnstreamer
+```
+
+### Clean build with pdebuild (Ubuntu 18.04)
```

Comment:
Ubuntu 16.04 and higher?

Reply:
16.04 is not available for now. But we will.